src/framework/tooltip/_tooltip.scss

```diff
@@ -0,0 +1,52 @@
+
+$arrowSide: 6px;
+$bgColor: $white;
+$borderColor: $componentStrokeColor;
+
+.tooltipWrapper {
+  display: inline-block;
+  position: relative;
+}
```

Comment:
Needs a `line-height: 0;` to prevent calculating the position on wrong premises.